os-ui/src/pages/CourseModulePage/Subcomponent/CourseModuleSidebar/CourseModuleSidebar.tsx

```diff
@@ -32,12 +32,13 @@ const CourseModuleSidebar = () => {
   const location = useLocation();
   const currentPath = location.pathname;
   let isbtnClicked = false;
-  if (currentPath === `/userCourse/modulOverview/${courseId}/discussionForum`) isbtnClicked = true;
-  if (currentPath === `/userCourse/modulOverview/${courseId}/discussionForum/AskMentor`) isbtnClicked = true;
-  if (currentPath === `/userCourse/modulOverview/${courseId}/discussionForum/AskPeers`) isbtnClicked = true;
-  if (currentPath === `/userCourse/modulOverview/${courseId}`) isbtnClicked = false;
-  const itemBtnclickedStyle = isbtnClicked ? { textDecoration: 'none', color: '#333941', fontWeight: '900' } : { textDecoration: 'none', color: '#333941' };
-  const itemBtnclickedRevStyle = isbtnClicked ? { textDecoration: 'none', color: '#333941' } : { textDecoration: 'none', color: '#333941', fontWeight: '900' };
+  const currentBaseUrl = `/userCourse/modulOverview/${courseId}`;
+  if (currentPath === `${currentBaseUrl}/discussionForum`) isbtnClicked = true;
+  if (currentPath === `${currentBaseUrl}/discussionForum/AskMentor`) isbtnClicked = true;
+  if (currentPath === `${currentBaseUrl}/discussionForum/AskPeers`) isbtnClicked = true;
+  if (currentPath === currentBaseUrl) isbtnClicked = false;
```

Comment:
I think instead of this "if" chain you can use a "switch" block, this way would be more optimal.